tools/src/launcher/Main.hs

```diff
@@ -442,13 +462,9 @@ runUpdater ndbp (path, args, runnerPath, mUpdateArchivePath) = do
 
 runUpdaterProc :: HasConfigurations => FilePath -> [Text] -> M ExitCode
 runUpdaterProc path args = liftIO $ do
-    let cr = (Process.proc (toString path) (map toString args))
-                 { Process.std_in  = Process.CreatePipe
-                 , Process.std_out = Process.CreatePipe
-                 , Process.std_err = Process.CreatePipe
-                 }
+    let cr = createProc Process.CreatePipe path args
     phvar <- newEmptyMVar
-    system' phvar cr mempty
+    system' phvar cr mempty EWallet
```

Comment:
Hm, updater is not wallet, it's another executable. Perhaps it makes sense to add `EUpdater`.